.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
 pkg/*.gem
+Gemfile.lock
```

Comment:
and no I don't want to discuss it or be pointed to the old blog post from mr catz ;)